app/src/main/java/com/quockhanhng/training/calculator/Utils.kt

```diff
@@ -0,0 +1,11 @@
+package com.quockhanhng.training.calculator
+
+const val ACTION_STABLE = 0
+const val ACTION_ADD = 1
+const val ACTION_SUBTRACT = 2
+const val ACTION_MULTIPLY = 3
+const val ACTION_DIVIDE = 4
+const val ACTION_EQUAL = 5
+
+const val TYPE_INPUT = 0
+const val TYPE_RESULT = 1
```

Comment:
Chia ra làm 2 object cho dễ quản lý 
```suggestion
object Actions {
    const val STABLE = 0
    const val ADD = 1
    const val SUBTRACT = 2
    const val MULTIPLY = 3
}

object Types {
    const val INPUT = 0
    const val RESULT = 1
}
```